src/databases/CGNS/CGNSCommonPluginInfo.C

```diff
@@ -44,14 +62,42 @@ avtDatabase *
 CGNSCommonPluginInfo::SetupDatabase(const char *const *list,
                                    int nList, int nBlock)
 {
+    avtDatabase *db = NULL;
```

Comment:
I can't recall if this is ever an issue but if xml2info is used to regen, will this custom code survive? I also can't recall if we added ability to indicate the existence of *custom* code in generated files in the `.xml` file to warn about this. If so, I didn't see that flag in the `.xml` file.

The above may all be me blowing smoke and confusing regular *attribute* objects (which are often re-gen'd) with database plugins which maybe are rarely if at all.